udata/features/territories/views.py

```diff
@@ -54,6 +54,33 @@ def redirect_town(code):
                             level='commune', code=code))
 
 
+@blueprint.route('/territory/town/<code>/', endpoint='town2')
```

Comment:
If the `endpoint` is not significant (ie. not called in other functions), just omit it. There is a sane default to the function/method name.